### ISSUE TITLE
Unity game engine 

### ISSUE THREAD

**Issue by Icezman001** [LutsenkoKirill/AlphaVantage.Net#23]
I'm trying to import the dll's into the Unity game engine but having quite a bit of trouble.

Is it possible to use this library in Unity? (trying to create a visual ministory which uses realtime stock prices).

If anyone knows, could they share how it could work?

I tried this method from another github package but it didn't seem to work:
https://anglesharp.github.io/docs/Questions.html#how-to-use-anglesharp-in-unity

**Comment by LutsenkoKirill** [comment]
@Icezman001  it was never been tested in Unity

**Comment by Icezman001** [comment]
> @Icezman001 it was never been tested in Unity

Yes I thought so. I'm willing to give it a proper test but I've been having issues with errors surrounding the .dll's.
I imagine the process to get it to work in Unity would be the same as other C# projects.
I'll try to figure it out, in the meantime if anyone has any pointers, please reply to this thread

**Comment by LutsenkoKirill** [comment]
@Icezman001 also feel free to simply download the code and include it in your project if you won't succeed with nuget package
